aimmo-game/tests/test_simulation/test_pickups.py

```diff
@@ -59,45 +58,38 @@ def test_health_increases(self):
     def test_serialise(self):
         self.assertEqual(self.pickup.serialise(), {'type': 'health', 'health_restored': 3})
 
-    # Test that the health pickup with a custom health gain value is applied correctly. We do not check
-    # for restraints (ie. less than 0 or more than 100) in the following tests:
-    def test_health_increases_custom_positive_5(self):
-        self.avatar.health = 10
+    def test_given_custom_health_pickup_increase_health_increases(self):
+        # Test 1 - increase of 5
+        self.assertEqual(self.avatar.health, 5)
```

Comment:
wait where is the avatar health set again?

Reply:
In the text fixture, setUp()